app/utils/cache/cache.go

```diff
@@ -0,0 +1,58 @@
+package cache
+
+import (
+	"sync"
+	"time"
+)
+
+type cacheMap struct {
```

Comment:
We technically don't need to encapsulate the fields in a struct anymore (as we are in a dedicated package) and we could just say:
```golang
var mutex sync.Mutex
var items = make(map[string]item)
```

(we can even skip the init function)